### PR TITLE
Retem task dialogue that everyone's already read

### PR DESCRIPTION
**Pull request by CAAAAAAAAAAAW**
I did not check for spelling errors very closely

### CODE REVIEW

**Comment by DabirSA** [comment]
Could you also look at already-translated task dialogue for a few characters, just to make the voices on the old stuff congruent with what you've done here? The ones I'd like you to look at are Lynce, Rasta, Tucano and Wallace. The old Wallace task will definitely need to be rewritten to match the voice you've given him here.

File: Translated/Dialog/Dana/ssn_014440.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name_7422#0,"""Dana"""
+ssn_014440_0001#0,"""Welcome to Retem City!"""
+ssn_014440_0002#0,"""I'm Dana, one of the lookouts here!"""
+ssn_014440_0003#0,"""If the enemy attacks, you'll know from us immediately. You'll be fighting then, too."""
```
Comment (DabirSA) [inline Translated/Dialog/Dana/ssn_014440.csv:4]: Is this definitely "fighting then" or should it be "fighting them"

Comment (CAAAAAAAAAAAW) [reply Translated/Dialog/Dana/ssn_014440.csv:4]: n, but they'd mean the same thing if you wanted m instead

File: Translated/Dialog/Rasta/ssn_014355.csv [hunk 2]
```diff
@@ -1,10 +1,10 @@
 name_7719#0,"""Ratel"""
-ssn_014355_0010#0,"""Hmm, just what could be up with this place?<br>Huh? You've been looking for me?"""
-ssn_014355_0020#0,"""So Rasta sent you, did she? I'm sorry if I made her worry. I was investigating these Ore veins, since I was outside the City already."""
-ssn_014355_0030#0,"""As you can see, I'm in no trouble here.<br>But my partner, though..."""
+ssn_014355_0010#0,"""Oh, what are you doing all the way out here?<br>Huh? You're looking for me?"""
```
Comment (DabirSA) [inline Translated/Dialog/Rasta/ssn_014355.csv]: For this line, "you've been" seems more accurate to the situation. You aren't currently looking for someone if you've just found them.

**Pull request merged**